components/places/src/bookmark_sync/incoming.rs

```diff
@@ -130,12 +96,13 @@ impl<'a> IncomingApplicator<'a> {
                 None
             }
         };
+        let unknown_fields = serde_json::to_string(&b.unknown_fields)?;
```

Comment:
How do you feel about writing a null if the map is empty? (And ditto for other items below)